src/main/java/Figure.java

```diff
@@ -0,0 +1,28 @@
+import java.util.Random;
+
+public class Figure {
+    public String drawFigure() {
+        return "фигура";
+    }
+
+    public double squareFigure() {
+        return 0.0;
+    }
+
+    public ColorFig colorFigure() {
+        return ColorFig.values()[new Random().nextInt(ColorFig.values().length)];
+    }
+
+    public FigureEnum[] arrayFigure() {
+        FigureEnum[] array = new FigureEnum[3 + (int) Math.random() * 20];//3+ обавила, чтобі не відавало массив 0
```

Comment:
```suggestion
        FigureEnum[] array = new FigureEnum[3 + (int) Math.random() * 20];//3+ обавила, чтобі не відавало массив 0
```
try to avoid comments in your code

Reply:
Or at least use English. This is a good practice